src/vivarium_public_health/disease/state.py

```diff
@@ -201,6 +199,7 @@ def setup(self, builder):
         disability_weight_data = get_disability_weight_func(self.cause, builder)
         self.prevalence_data = builder.lookup.build_table(get_prevalence_func(self.cause, builder))
         self._dwell_time = get_dwell_time_func(self.cause, builder)
+        self.randomness = builder.randomness.get_stream(f'{self.state_id}_prevalence_event_time')
```

Comment:
Give this a more descriptive name.  Since this class is two levels down the inheritance hierarchy, if we add a randomness stream to `BaseDiseaseState` or `State`, we'd likely override it without noticing here and then end up using the same random numbers for two potentially unrelated things.